workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectWorldCommandSystem.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Collections;
+using Unity.Entities;
+
+#pragma warning disable 649
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    // ReSharper disable once ClassNeverInstantiated.Global
+    public class GameObjectWorldCommandSystem : ComponentSystem
```

Comment:
Can we not use the existing stuff?

Reply:
I misread this file. If this is the system that detects you have callbacks registered then this is fine. In fact it's how I'd expect all such systems to work.